Web/Edubase.Services/Nomenclature/NomenclatureService.cs

```diff
@@ -43,6 +43,7 @@ public static string GetEstablishmentsPluralName(GT groupType, eTextCase textCas
                 case GT.MultiacademyTrust:
                 case GT.SingleacademyTrust:
                 case GT.SchoolSponsor:
+                case GT.SecureSingleAcademyTrust:
```

Comment:
This code references SAT - should Secure SAT be included also?

https://github.com/DFE-Digital/get-information-about-schools/blob/secure_16-19_groups/Web/Edubase.Web.UI/Areas/Groups/Views/Group/Partials/Details/_GenericGroupDetails.cshtml#L26-L32

```csharp
@if (Model.Group.GroupTypeId.OneOfThese(eLookupGroupType.MultiacademyTrust, eLookupGroupType.SingleacademyTrust, eLookupGroupType.SchoolSponsor))
{
    <div class="govuk-summary-list__row">
        <dt id="group-identifier-key" class="govuk-summary-list__key">Group identifier (ID)</dt>
        <dd id="group-identifier-value" class="govuk-summary-list__value">@(Model.Group.GroupId.Clean() ?? "Not recorded")</dd>
    </div>
}
```

Reply:
This is one of the fields that's not displayed for secure academy groups